### PR TITLE
Hotfix/277 287 text grammar

### PR DESCRIPTION
**Pull request by ArturTrainee**

### CODE REVIEW

File: backend/core/build.gradle [hunk 1]
```diff
@@ -59,5 +59,5 @@ test {
 flyway {
 	url = 'jdbc:postgresql://localhost:5432/knewless'
 	user = 'postgres'
-	password = 'admin'
+	password = 'postgre'
```
Comment (MaximGoncharukSW) [inline backend/core/build.gradle]: Don't change default pass in repo

File: backend/core/src/main/resources/application.properties [hunk 2]
```diff
@@ -14,7 +14,7 @@ fs.video_url = http://localhost:5000/
 #-------------------------
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.datasource.username=postgres
-spring.datasource.password=admin
+spring.datasource.password=postgre
```
Comment (MaximGoncharukSW) [inline backend/core/src/main/resources/application.properties]: Don't change default pass in repo

File: backend/core/src/main/resources/application.yml [hunk 3]
```diff
@@ -4,12 +4,16 @@ spring:
...
+            clientSecret: 0IUhkQ2m27xutc49vYmfIPUn
+            redirectUri: "{baseUrl}/oauth2/callback/{registrationId}"
             scope:
               - email
               - profile
           facebook:
-            redirectUri: "https://localhost{basePort}/oauth2/callback/{registrationId}"
+            clientId: 575453373121776
+            clientSecret: 41655959e0f13d393a44575143044425
+            redirectUri: "{baseUrl}/oauth2/callback/{registrationId}"
```
Comment (MaximGoncharukSW) [inline backend/core/src/main/resources/application.yml]: Don't commit credentials

File: backend/core/src/main/resources/application.yml [hunk 4]
```diff
@@ -21,5 +25,4 @@ spring:
 app:
   auth:
     tokenSecret: mdzLa0RXxQX8mL9hFjctK9hCTDW4BQXHZXQZPPAKdD0LR6K0rB9rJrb6rM1otpWGXVABmxH0n4G1oFkp5Z03fGMXpvi3yeMq9jgxpnnNU6vP0j2UneEdm8Me1JK9qEQV
-    tokenExpirationMsec: 86400000
-    refreshExpirationMsec: 2592000000
+    tokenExpirationMsec: 864000000
```
Comment (MaximGoncharukSW) [inline backend/core/src/main/resources/application.yml]: Return back refreshTokenExp

File: backend/core/src/main/resources/application.properties [hunk 5]
```diff
@@ -14,7 +14,7 @@ fs.video_url = http://localhost:5000/
 #-------------------------
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.datasource.username=postgres
-spring.datasource.password=admin
+spring.datasource.password=demo
```
Comment (AndersenGans) [inline backend/core/src/main/resources/application.properties]: u sure?

File: backend/core/build.gradle [hunk 6]
```diff
@@ -59,5 +59,5 @@ test {
 flyway {
 	url = 'jdbc:postgresql://localhost:5432/knewless'
 	user = 'postgres'
-	password = 'admin'
+	password = 'demo'
```
Comment (AndersenGans) [inline backend/core/build.gradle]: u sure?

**Pull request merged**